src/components/v2/V2Project/V2PayButton.tsx

```diff
@@ -11,14 +11,18 @@ import { V2_CURRENCY_ETH, V2_CURRENCY_USD } from 'utils/v2/currency'
 import PayWarningModal from 'components/shared/PayWarningModal'
 import useWeiConverter from 'hooks/WeiConverter'
 
+import { CurrencyOption } from 'models/currencyOption'
+
+import { V2CurrencyOption } from 'models/v2/currencyOption'
+
 import V2ConfirmPayOwnerModal from './V2ConfirmPayOwnerModal'
 
 export default function V2PayButton({
   payAmount,
   payInCurrency,
 }: {
   payAmount: string
-  payInCurrency: number
+  payInCurrency: CurrencyOption // TODO make the V2CurrencyOption
```

Comment:
I'm sure there exists a better solution to this, but this change is still a net-improvement.